_opensearch/install/rpm.md

```diff
@@ -5,154 +5,472 @@ parent: Install OpenSearch
 nav_order: 51
 ---
 
-# RPM
-
-The RPM Package Manager (RPM) installation provides everything you need to run OpenSearch inside Red Hat or Red Hat-based Linux Distributions.  
+{% comment %}
+The following liquid syntax declares a variable, major_version_mask, which is transformed into "N.x" where "N" is the major version number. This is required for proper versioning references to the Yum repo.
+{% endcomment %}
+{% assign version_parts = site.opensearch_major_minor_version | split: "." %}
 
-RPM supports CentOS 7 and 8, and Amazon Linux 2. If you have your own Java installation and set `JAVA_HOME` in your terminal application, macOS works, as well.
+# RPM
 
-There are two methods for installing OpenSearch on RPM: 
+Installing OpenSearch using RPM Package Manager (RPM) simplifies the process considerably compared to the [Tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/) method. Several technical considerations, such as the installation path, location of configuration files, and the creation of a service managed by `systemd`, as examples, are handled automatically by the package manager.
 
-## Manual method
+Generally speaking, installing OpenSearch from the RPM distribution can be broken down into a few steps:
 
+1. **Download and install OpenSearch.**
+   - Installation is performed by manually installing from an RPM distribution, or by creating a local YUM repository and installing with `yum`.
```

Comment:
suggestion for active voice:

You can either manually install an RPM distribution, or you can create a local YUM repository and then install with `yum` command.

Reply:
Thanks! Will reword.